android/app/src/main/java/io/peggyjo/peggymeter/HistoryTextFragment.java

```diff
@@ -31,6 +38,37 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         dataController.addMoodListener(this);
         // Inflate the layout for this fragment
         View view = inflater.inflate(R.layout.fragment_history_text, container, false);
+        ListView table = view.findViewById(R.id.LOG_ENTRIES_CONTAINER);
```

Comment:
This ListView can be a member so we don't have to call findViewById twice: here and down in renderLog.